ghcide/src/Development/IDE/GHC/Compat/Core.hs

```diff
@@ -943,11 +889,7 @@ unrestricted = id
 
 mkVisFunTys :: [Scaled Type] -> Type -> Type
 mkVisFunTys =
-#if __GLASGOW_HASKELL__ == 808
-  mkFunTys
-#else
   TcType.mkVisFunTys
```

Comment:
Is there potential for inlining code (e.g. here)?

Reply:
yep, that one can go!